lua/toggleterm.lua

```diff
@@ -6,6 +6,8 @@ local constants = require("toggleterm.constants")
 local colors = require("toggleterm.colors")
 
 local terms = require("toggleterm.terminal")
+local ui = require("toggleterm.ui")
```

Comment:
@zeertzjq in order to minimize startup time lua plugins should defer requiring as much as possible till the point where something is needed. I've purposely required things like config inside the functions so that as much as possible requires don't happen at the top of the file since that means just by a user requiring the setup function they basically start a nested chain of requires.

Reply:
Also tbh it just adds more visual noise to the PR to have changes that aren't related explicitly to what this PR is trying to achieve so it will be harder for me to properly review.